enterprise/query-logging/src/test/java/org/neo4j/kernel/impl/query/QueryLoggerIT.java

```diff
@@ -350,7 +354,7 @@ public void queryLogRotation() throws Exception
                               .map( this::readAllLinesSilent )
                               .flatMap( Collection::stream )
                               .collect( Collectors.toList() );
-        assertThat( "Expected log file to have at least one log entry", loggedQueries.size(), lessThan( 200 ) );
+        assertThat( "Expected log file to have at least one log entry", loggedQueries.size(), lessThanOrEqualTo( 202 ) );
```

Comment:
Why do we check `lessThan`? Is it not more interesting to check `greaterThan` if we want to make sure that some number of queries are logged?
Failure message could use an update as well.

Reply:
logging and rotation is async and you do not know have many of your queries where logs in which file and you can't rely on how much queries will be in a file that new rotation policy: "CALL dbms.setConfigValue('" + log_queries_max_archives.name() + "','1')" will keep.

Reply:
Because this test makes an online configuration change that will reduce the number of query log files that we keep. This change must take effect for the log files that exists at that time, so we must end up with _fewer_ log files than we otherwise would.

Reply:
Aah, yes! Thanks :D